browser-test/src/admin_application_statuses.test.ts

```diff
@@ -9,11 +9,22 @@ import {
   AdminPrograms,
   userDisplayName,
   AdminProgramStatuses,
+  enableFeatureFlag,
+  gotoEndpoint,
+  disableFeatureFlag,
 } from './support'
 import {Page} from 'playwright'
+import EmptyFunction = jest.EmptyFunction
 
-// TODO(#3071): Re-enable when the feature flag is controllable in tests.
-describe.skip('view program statuses', () => {
+const describeIf = (
```

Comment:
If we expect this to be a long-term solution, I'd suggest moving the definition to something in `support`.

Reply:
removed